cli/cli/commands/port/print/print.go

```diff
@@ -42,7 +48,14 @@ var PortPrintCmd = &engine_consuming_kurtosis_command.EngineConsumingKurtosisCom
 	LongDescription:           "Get information for port using port id",
 	KurtosisBackendContextKey: kurtosisBackendCtxKey,
 	EngineClientContextKey:    engineClientCtxKey,
-	Flags:                     []*flags.FlagConfig{},
+	Flags: []*flags.FlagConfig{
+		{
+			Key:     formatFlagKey,
+			Usage:   "Allows selecting what pieces of port are printed, using comma separated values (examples: 'ip', 'number', 'protocol,ip'). Default 'protocol,ip,number'.",
```

Comment:
Can we make these named constants, that correspond to the values down below?

Reply:
In all the other commands in the CLI do not have their usage string extracted to constants: https://github.com/kurtosis-tech/kurtosis/blob/main/cli/cli/commands/run/run.go#L109-L113

I ended up going for keeping the pattern as we have it currently.

Reply:
Make sense, will do